packages/adapter/test/adapter.spec.ts

```diff
@@ -51,6 +51,29 @@ fixture.test(
   }
 )
 
+fixture = createFixture({
+  isolated: true,
+  download: {
+    flask: true,
+  },
+  snap: {
+    id: SNAP_ID,
+    version: SNAP_VERSION,
+  },
+  // Rainbow wallet extension id, once https://github.com/hugomrdias/filsnap-testing/pull/30 lands this can be uncommented
```

Comment:
Add this back once https://github.com/hugomrdias/filsnap-testing/pull/30 is complete